changes/4a4022c654b41f3198f9fc3a1d340a0e.yaml

```diff
@@ -0,0 +1,6 @@
+---
+desc: Updated Storm package onload/inits to only run on Cortex leaders.
```

Comment:
```suggestion
desc: Updated Storm package onload/inits to always run directly on the leader even when a Storm query pool is in use.
```

Reply:
tweaked to avoid confusion about the current implementation running on leaders and mirrors